source/includes/documentation/_vue_js_elements.md

```diff
@@ -2693,12 +2693,27 @@ For examples of usage see: [JDI vuetify page tests for Skeleton Loader](https://
 
 The `v-parallax` component creates a 3D effect that makes an image appear to scroll slower than the window. A parallax causes a shift in a background image when the user scrolls the page.
 
+![v-parallax example](../../images/vuetify/parallax.png)
+
+```html
+<div class="pa-4 v-sheet theme--light rounded">
+   <div class="v-parallax" file="v-parallax/usage" style="height: 500px;">
+     <div class="v-parallax__image-container"><img src="https://cdn.vuetifyjs.com/images/parallax/material.jpg" alt="" class="v-parallax__image" style="display: block; opacity: 1; transform: translate(-50%, 219px);">
+     </div>
+       <div class="v-parallax__content"></div>
+    </div>
+</div>
+
+```
+
+v-parallax element has following methods:
+
 |Method | Description | Return Type
```

Comment:
Please,
1) add framework version
2) in a documentation near highlighted comment please also emphasize a Selenium locator findby 
![image](https://user-images.githubusercontent.com/55189259/212971030-e184e6ca-1e0e-4b50-9ef6-18219f3aaf76.png)